core/src/iam/verify.rs

```diff
@@ -286,7 +274,7 @@ pub async fn token(kvs: &Datastore, session: &mut Session, token: &str) -> Resul
 				// If the access type is Record, this is record access
 				// Record access without an "id" claim is only possible if there is an AUTHENTICATE clause
 				// The clause can make up for the missing "id" claim by resolving other claims to a specific record
-				AccessType::Record(at) => match at.authenticate {
+				AccessType::Record(at) => match de.authenticate.clone() {
```

Comment:
(same) Do we have to clone each time we test this condition?
Couldn't use a ref, and clone what we need once the condition is met?

Reply:
Good catch! Addressed in [d4d2a4b](https://github.com/surrealdb/surrealdb/pull/4385/commits/d4d2a4bd995b1bdc0a4d7752961755c6b3c72bf2).